cfn-resources/cloud-backup-schedule/mongodb-atlas-cloudbackupschedule.json

```diff
@@ -216,7 +216,8 @@
   },
   "additionalProperties": false,
   "required": [
-    "Profile"
+    "Profile",
+    "AutoExportEnabled"
```

Comment:
Is this a breaking change or was it always actually required and just threw and error if not set?

If its required, can you also check it's specified in the example?

Reply:
see response here: https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/573#issuecomment-1564589260

Reply:
@colm-quinn yes this is on the example, is a "braking change" but if the user does not provide this parameter he will het a null pointer, with this he will get an actual validation